test/pjrt/test_runtime.py

```diff
@@ -83,6 +77,7 @@ def test_pjrt_default_device(self, env_vars, expect_using_pjrt):
       if expect_using_pjrt:
         self.assertIn(xr.device_type(), ['CPU', 'GPU', 'TPU'])
       else:
+        print("XRT has been deprecated, please set PJRT_DEVICE")
```

Comment:
ditto

Reply:
Thanks for pointing, will delete

Reply:
Thanks, updated in https://github.com/pytorch/xla/pull/5202